samples/sampler/stories/default/Panel.js

```diff
@@ -13,13 +13,15 @@ export default {
 };
 
 export const _Panel = (args) => (
-	<Panel>
-		<Header
-			subtitle={args['subtitle']}
-			title={args['title']}
-		/>
-		<Button>Click me</Button>
-	</Panel>
+	<div style={{minHeight: '400px'}}>
```

Comment:
`import ri from '@enact/ui/resolution';
<div style={{minHeight: ri.scaleToRem(400px)}}>
`

same pattern for every sample